routers/web/repo/projects.go

```diff
@@ -444,12 +445,9 @@ func UpdateIssueProject(ctx *context.Context) {
 		return
 	}
 
-	projectID := ctx.FormInt64("id")
+	projectIDs, _ := base.StringsToInt64s(strings.Split(ctx.FormString("id"), ","))
```

Comment:
Maybe the query parameter name should be renamed to `ids`.

Reply:
id cannot be renamed to ids, because the shared updateToBackend function generates